### ISSUE TITLE
Detecting if cwd doesn't contain a Nuxt project

### ISSUE THREAD

**Issue by adhamfarrag** [nuxt/cli#203]
Default behavior for using the CLI in a directory without `nuxt.config.ts` is installing, then creating the Nuxt config file and adding the module. Can we check if `nuxt.config.ts` exists first ? 

Thinking behavior should be either showing a YES/NO question to create it and proceed? Or, just killing the process.

**Comment by adhamfarrag** [comment]
@pi0 Happy to do the PR. :)

**Comment by pi0** [comment]
Certainly! PR more than welcome to add this guard. I think we need to check either of `pages/` dir, `nuxt.config` or `app.vue` files exist.

**Comment by adhamfarrag** [comment]
I think it should be one rule checking if it's Nuxt project or not (searching for config file) and to be added to the add/module commands.

**Comment by OrbisK** [comment]
I think this was also solved with https://github.com/nuxt/cli/pull/362
